manifests/nodeExporter-prometheusRule.yaml

```diff
@@ -134,6 +134,28 @@ spec:
       for: 1h
       labels:
         severity: critical
+
+    - alert: NodesMemoryFillingUp
+      annotations:
+        description: Average nodes memory utilization is {{ printf "%.2f" $value }}% and is filling up.
+        runbook_url: https://runbooks.prometheus-operator.dev/runbooks/node/nodesmemoryfillingup
```

Comment:
not blocking: It would be awesome if you could also contribute with a PR adding this to the runbooks repo